admin/admin-service/src/main/resources/application-local1.properties

```diff
@@ -6,6 +6,7 @@ auth.server.user-add-password-url=https://api-internal.dev.mosip.io/v1/authmanag
 mosip.kernel.config.server.file.storage.uri=https://qa3.mosip.net/config/print/mz/qa3-1.1.5/
 mosip.admin.identityMappingJson=identity-mapping.json
 mosip.admin.applicant-details.exposed-identity-fields=dob,applicantPhoto
+mosip.admin.doccodes-cleanup.fixed-rate=24
```

Comment:
Could you add a comment to specify the value unit? and also what codes are cleaned.